.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+*.log
```

```suggestion
logs/*
```
and add an empty line

---

done